habitat_baselines/rl/ppo/utils.py

```diff
@@ -422,4 +425,72 @@ def ppo_args():
         nargs=argparse.REMAINDER,
         help="Modify config options from command line",
     )
+    parser.add_argument(
+        "--tensorboard-dir",
+        type=str,
+        default="tb_train",
+        help="path to logging dir for Tensorboard",
+    )
     return parser
+
+
+def frames_to_tb_video(video_name, step_idx, frames, writer, fps=10):
+    """
+    write video into tensorboard from images frames
+    :param video_name: name of video string
```

Comment:
Be consistent in writing docstring, I think this does not conform to the format used elsewhere.

Reply:
Thanks for noticing that. I found that **Settings | Tools | Python Integrated Tools | Docstring format** to **Google** in PyCharm will output correct Docstring.